charts/core/traefik/templates/_args.tpl

```diff
@@ -55,14 +55,17 @@ args:
   - "--providers.kubernetesingress.labelSelector={{ .Values.providers.kubernetesIngress.labelSelector }}"
   {{- end }}
   {{- end }}
-  {{- if and .Values.rbac.enabled .Values.rbac.namespaced }}
+  {{- if .Values.rbac.enabled }}
```

Comment:
We do not use namespaced rbac afaik
and you definately don't want namespaced CRD's for sure.

Reply:
I'm not sure what these namespaces commands have to do with rbac, but they simplified my traefik deployments by only having middlewares and routes that were relevant to each deployment.

Tomorrow I will try to find out more and have a suggestion for how to move forward.

Reply:
Please just leave these changes outside of your PR completely.
Its not needed to mess with rbac pedmissions and I will not merge it either.